dmscripts/export_framework_results_reasons.py

```diff
@@ -194,6 +161,13 @@ def create_row(self, record):
             contact_details(record)
 
 
+def get_questions_numbers_from_framework(framework_slug, content_loader):
+    return {
```

Comment:
Yep this needs  `content_loader.load_manifest(framework_slug, 'declaration', 'declaration')` before we do the `get_manifest`, but then we get `AttributeError: 'ContentManifest' object has no attribute 'questions'`